s3-vfs/src/main/java/org/pentaho/amazon/s3/S3FileOutputDialog.java

```diff
@@ -1158,7 +1158,7 @@ public void widgetSelected( SelectionEvent e ) {
               BaseMessages.getString( BASE_PKG, "System.FileType.AllFiles" ) };
 
           if ( StringUtils.isEmpty( wFilename.getText().trim() ) ) {
-            wFilename.setText( "s3n://s3n/" );
+            wFilename.setText( "s3a://" );
```

Comment:
Is it really "s3a" or should it stay as "s3n" like on lines 323 and 1478?

Reply:
yes, the s3a is the new default format